Something/Service/Services/Company/CompanyService.cs

```diff
@@ -17,17 +17,17 @@ public CompanyService(ApplicationDbContext dbContext)
             _dbContext = dbContext;
         }
 
-        public IEnumerable<CompanyViewModel> GetCompanies(Guid userId)
+        public IEnumerable<CompanyVM> GetCompanies(Guid userId)
         {
             var companies = _dbContext
                                     .CompanyUsers
                                     .Where(x => x.UserId == userId)
                                     .Select(x => x.Company).ToList();
```

Comment:
Is the lazy loading ON by default ?

Reply:
Sorry what I meant was i like to know what related data I load in a query by using Includes. https://docs.microsoft.com/en-us/ef/core/querying/related-data
It helps to give a clear picture that you're not wrongly loading unrelated data.

Reply:
Good point. Will update. Reference - http://foreverframe.net/when-use-include-with-entity-framework/

Reply:
These 2 comment changes I will do in a different PR, and merge this one.

Reply:
Done here - https://github.com/yrshaikh/Something/pull/38